### PR TITLE
feat: New driver `did:algo`

### PR DESCRIPTION
**Pull request by bcessa**
Documentation is available at:
https://github.com/algorandfoundation/did-algo/blob/main/DRIVER.md

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -287,3 +287,7 @@ services:
     image: kataru/content-did-driver:latest
     ports:
       - "8148:8888"
+  driver-did-algo:
+    image: ghcr.io/aidtechnology/algoid-resolver:latest
```
Comment (dif-admin) [inline docker-compose.yml:291]: Replace with tagged version

**Review by dif-admin — CHANGES_REQUESTED** [review]
Thanks for this PR @bcessa! 
One small update: we no longer support use of the "latest" tag; can you please replace this?

**Comment by bcessa** [comment]
> Thanks for this PR @bcessa! One small update: we no longer support use of the "latest" tag; can you please replace this?

Thanks for pointing it out @dif-admin 👏🏼 we added the image tag (`0.6.0`) now.

**Review by peacekeeper — CHANGES_REQUESTED** [review]
This seems to make some changes to other driver configs, replacing `"` with `'` characters in `application.yml`. Could you change the PR to only add your driver, without making changes to other drivers?

**Comment by bcessa** [comment]
> This seems to make some changes to other driver configs, replacing `"` with `'` characters in `application.yml`. Could you change the PR to only add your driver, without making changes to other drivers?

You are right @peacekeeper was unintended, should be reverted now.

**Review by peacekeeper — APPROVED** [review]
Thanks looks good!

**Pull request merged**